event-driven-async-generated/src/event.rs

```diff
@@ -100,7 +100,7 @@ impl EventTask {
     ///
     /// This function emits tick events at a fixed rate and polls for crossterm events in between.
     async fn run(self) -> color_eyre::Result<()> {
-        let tick_rate = Duration::from_secs_f64(TICK_FPS);
+        let tick_rate = Duration::from_secs_f64(1 / TICK_FPS);
```

Comment:
I'm guessing this should be 1.0, not 1 here...

Reply:
Yep, you're right. Can't divide float by int.